### DISCUSSION TITLE
R1's comments

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#292] [General]
Since all of R1's comments are closely related (IMO), I think it would be best for 2 or 3 of us to work on all of them kind of at once. I went ahead and assigned myself to all of them, but it would be great if one or two others would join me.  I propose that this small group try to meet sometime in the next few weeks having recently read through the manuscript.  We can discuss a plan of attack, then I could take a first pass at revising before we reconvene.  Anyone want to help out?

**Comment by emmajhudgins** [comment]
I'm up for it! Sorry i missed this when looking through the issues

**Comment by pedrohbraga** [comment]
Hi, @Aariq! I also missed this, but I am also in for it!

@fhillemann also mentioned that she would be able to put in some time next week!

I will send us a poll to see if we can find a common availability and meet to get these ones done!

**Comment by pedrohbraga** [comment]
I sent the poll through email, but you can also find it [here](https://www.when2meet.com/?17950945-vyhmz).

**Comment by Aariq** [comment]
Notes from our meeting: https://hackmd.io/ICM9eNqkQFyLBH2_9G3i_Q?both

**Comment by Aariq** [comment]
Should the Peer Review section be merged with the Writing Manuscripts section?
